lib/Db/PromptMapper.php

```diff
@@ -182,6 +182,26 @@ public function deleteUserPrompts(string $userId): void {
 		$qb->resetQueryParts();
 	}
 
+	/**
+	 * Delete user prompts by type
+	 * @param string $userId
+	 * @param int $type
+	 * @return void
+	 * @throws Exception
+	 */
+	public function deleteUserPromptsByType(string $userId, int $type): void {
+		$qb = $this->db->getQueryBuilder();
+		$qb->delete($this->getTableName())
+			->where(
+				$qb->expr()->eq('user_id', $qb->createNamedParameter($userId, IQueryBuilder::PARAM_STR))
+			)
+			->andWhere(
+				$qb->expr()->eq('type', $qb->createNamedParameter($type, IQueryBuilder::PARAM_INT))
+			);
+		$qb->executeStatement();
+		$qb->resetQueryParts();
```

Comment:
not necessary, $qb will be out of scope after this statement

Reply:
Thanks! Fixed.